Clinical-Genomics/cg
# update bootstrap to 4.1.3

How to test:
0. Install branch on stage on clinical-db
1. Open clinical-api-stage in a browser-window
2. Open clinical-api-stage in another browser-window
3. Compare the two browsers windows
4. Go through every clinical-api-stage Invoice view and compare

Expected outcome:
    Every invoice view looks exactly the same
